File: SecondYear/Parth27904_ParthAgarwal_2226CSE(AI&ML)1015_2/Week_2/count_Digits.java
public class count_Digits {
    public static int countDigits(int n){
        int temp = n;
        int digit = 0;
        int count = 0;
        while(temp!=0){
            digit = temp%10;
            if(digit != 0 && n%digit == 0){
                count++;
            }
            temp = temp/10;
        }
        return count;        
    }
}

File: SecondYear/Adityacheeze_AdityaMohanGupta_2226CSE(AI&ML)1018_2/Week_2/Count Digits.cpp
int countDigits(int n){
	// Write your code here.
	int temp = n;
	int count = 0;
	while(temp != 0){
		int digit = temp % 10;
		if(digit != 0 && n % digit == 0){
			count++;
		}
		temp = temp / 10;
	}	
	return count;
}